internal/github/auth.go

```diff
@@ -0,0 +1,51 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License
+// 2.0; you may not use this file except in compliance with the Elastic License
+// 2.0.
+
+package github
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
I would remove usage of this package in favour of `fmt.Errorf` from the standard library. What do you think?

Reply:
I think I'ts a good idea given how `fmt.Errorf ` works since Go 1.13.